### PR TITLE
Homework #9

### PR DESCRIPTION
**Pull request by AndryNosach**
Done:
1. Used hibernate instead jdbc.
2. Fixed some front-end stuff.

### CODE REVIEW

File: Nosach/web-quiz-project/src/main/java/dao/QuizHibernateDAOImpl.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        session.save(quiz);
+        session.flush();
+        transaction.commit();
+        return quiz.getId();
+    }
+
+    @Override
+    public Quiz getQuiz(int id) {
+        Session session = sessionFactory.openSession();
+        Transaction transaction = session.beginTransaction();
```
Comment (dinamio) [inline Nosach/web-quiz-project/src/main/java/dao/QuizHibernateDAOImpl.java:36]: Find can be done without transaction

File: Nosach/web-quiz-project/src/main/java/dao/QuizHibernateDAOImpl.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        session.flush();
+        transaction.commit();
+        return quiz;
+    }
+
+    @Override
+    public List<Quiz> getAllQuizzies() {
+        Session session = sessionFactory.openSession();
+        Transaction transaction = session.beginTransaction();
+        Query query = session.createQuery("from Quiz");
```
Comment (dinamio) [inline Nosach/web-quiz-project/src/main/java/dao/QuizHibernateDAOImpl.java:47]: Same comment, as before

File: Nosach/web-quiz-project/src/main/java/entity/Answer.java [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
+
+@Entity
+@Table(name = "answers")
 public class Answer {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @Column(name = "answer")
```
Comment (dinamio) [inline Nosach/web-quiz-project/src/main/java/entity/Answer.java:13]: If column the same, you mustn't write @Column annotation. But it's more readable

**Review by dinamio — APPROVED** [review]
Looks good

File: Nosach/web-quiz-project/src/main/java/dao/QuizHibernateDAOImpl.java (hunk 1 above)
Comment (AndryNosach) [reply Nosach/web-quiz-project/src/main/java/dao/QuizHibernateDAOImpl.java:36]: Ok.

**Pull request merged**